app/models/cvp/person.rb

```diff
@@ -0,0 +1,66 @@
+# encoding: utf-8
+
+#  Copyright (c) 2012-2020, CVP Schweiz. This file is part of
+#  hitobito_cvp and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_cvp.
```

Comment:
we decided to get rid of those headers :) so only frozen_string_literal: true
also encoding: utf-8 is not required anymore